INSTALL.md

```diff
@@ -3,10 +3,10 @@ On Ubuntu 18.04:
 ```
 sudo apt-get update
 sudo apt-get install git cmake clang-6.0 llvm-6.0-tools zlib1g-dev bison flex libboost-test-dev libgmp-dev libmpfr-dev libyaml-cpp-dev libjemalloc-dev curl libtinfo-dev
-./install-rust
```

Comment:
can you explain this change? if it's just because you want to change the directory that the rust sources get placed in, I honestly think it makes more sense for it to be outside so that users don't delete it when they run `git clean -fxd`.

Reply:
I just moved after the `git clone` because the script is not available before you do that. I guess we can change further to make the script execute outside of the repo dir. Do you want me to do that?

Reply:
that makes sense, but yes, let's do it before the cd.